frontend/src/component/project/Project/Project.tsx

```diff
@@ -240,6 +240,7 @@ const Project = () => {
                         value={activeTab?.path}
                         indicatorColor="primary"
                         textColor="primary"
+                        variant="scrollable"
```

Comment:
Makes this part scrollable
<img width="480" alt="Screenshot 2022-12-06 at 13 50 20" src="https://user-images.githubusercontent.com/1394682/205921765-2dbfedef-851a-4b30-9fb9-4d18d33baf26.png">